File: problem/ZeroFilled_Subarrays/zeroFilledSubarray.c
long long zeroFilledSubarray(int *nums, int numsSize)
{
    long long cnt = 0, streak = 0;
    for (int i = 0; i < numsSize; i++)
    {
        if (nums[i] == 0)
        {
            streak++;
            cnt += streak;
        }
        else
        {
            streak = 0;
        }
    }
    return cnt;
}

File: problem/ZeroFilled_Subarrays/zeroFilledSubarray.py
class Solution:
    def zeroFilledSubarray(self, nums):
        ans = 0
        n = len(nums)
        curr = 0
        for i in range(n):
            if nums[i] == 0:
                curr += 1
            else:
                ans += curr * (curr + 1) // 2
                curr = 0
        ans += curr * (curr + 1) // 2
        return ans